apps/server/src/core/error/loggable/axios-error.loggable.spec.ts

```diff
@@ -12,6 +12,7 @@ describe(AxiosErrorLoggable.name, () => {
 			const axiosError: AxiosError = axiosErrorFactory.withError(error).build();
 
 			const axiosErrorLoggable = new AxiosErrorLoggable(axiosError, type);
+			axiosErrorLoggable.stack = 'mockStack';
```

Comment:
I think it's better to test for `axiosErrorLoggable.stack` in line 29 because it's more expressive.

Reply:
Then you can also delete this line. That's why I commented here. :)